### ISSUE TITLE
Java6 wrong download link in cask

### ISSUE THREAD

**Issue by Lenascha** [Homebrew/homebrew-cask#71095]
#### General troubleshooting steps

- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [x] I have retried my command with `--force`.
- [x] I ran `brew update-reset && brew update` and retried my command.
- [x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [x] I made doubly sure this is not a [checksum does not match](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md) error.

#### Description of issue

The download link given in the Java6 cask is no longer working. 
The homepage 'https://support.apple.com/kb/DL1572' link is still correct, however when downloading from there the given link is : https://updates.cdn-apple.com/2019/cert/041-88384-20191011-3d8da658-dca4-4a5b-b67c-26e686876403/JavaForOSX.dmg

I am sorry for creating a new issue, but I do not know enough about cask yet to fix it myself.
Thank you!

#### Command that failed

```
curl: (22) The requested URL returned error: 404 Not Found
```

#### Output of command with `--force --verbose --debug`

```
Not relevant.
```

#### Output of `brew cask doctor`

```
Not relevant.
```

#### Output of `brew tap`

```
Not relevant.
```

**Comment by robertleeblairjr** [comment]
I believe some maintainers are working on determining what they plan on doing with Java deprecated casks and the naming conventions. I submitted a PR for you to update the cask. If you have time, then check out the following link for setting up to contribute. It doesn't take long and you'll be able to help out with submitting these change requests. Take care.

[https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md](url)

**Comment by vitorgalvao** [comment]
Fixed in https://github.com/Homebrew/homebrew-cask-versions/pull/8021.

**Comment by robertleeblairjr** [comment]
> Fixed in [Homebrew/homebrew-cask-versions#8021](https://github.com/Homebrew/homebrew-cask-versions/pull/8021).

Thanks.
